bluesea_node:
  ros__parameters:
  #ROS2
    frame_id: "map"
    topic: "scan"
  #DATA#
    dev_id: -1
    raw_bytes: 3
    output_360: true
    output_scan: true
    output_cloud: false
    with_angle_filter: false
    min_angle: -3.14159
    max_angle: 3.14159
    # mask1: "0,3.14"
    # mask2: "-3,-2"
    hard_resample: false
    soft_resample: false
    with_checksum: true
    reversed: false
    min_dist: 0.1
    max_dist: 50.0
    error_circle: 3
    error_scale: 0.9
  #CONNECT#
    type: "vpc"
    port: "/dev/ttyACM0"
  #GET#
    uuid:  -1
  #SET#
    resample_res: -1.0
    with_confidence: -1
    with_smooth: -1
    with_deshadow: -1
    alarm_msg: -1
    rpm: -1
    direction:  -1
    ats: 1